test/remote/gateways/remote_element_test.rb

```diff
@@ -41,6 +42,12 @@ def test_successful_purchase_with_payment_account_token
     assert_equal 'Approved', response.message
   end
 
+  def test_successful_purchase_with_shipping_address
+    response = @gateway.purchase(@amount, @credit_card, @options.merge(shipping_address: address(address1: "Shipping")))
+    assert_success response
+    assert_equal 'Approved', response.message
+  end
```

Comment:
@davidsantoso I think that's the expected behavior; in fact it only returns the first line and zipcode of the billing address in the response as well. PR is updated with the address element fix.